src/ILLink.RoslynAnalyzer/ISymbolExtensions.cs

```diff
@@ -49,5 +51,38 @@ internal static bool TryGetAttributeWithMessageOnCtor (this ISymbol symbol, stri
 
 			return false;
 		}
+
+		internal static bool Contains<T, TComp> (ImmutableArray<T> list, T elem, TComp comparer)
```

Comment:
These functions aren't extending anything, we should put them in a new file.

Reply:
Now there is a file called ImmutableArrayOperations